test/test_script.py

```diff
@@ -390,6 +390,45 @@ def test_cmd_with_not_arguments(self):
         except ScriptError:
             self.fail("Function noargs raised ScriptError unexpectedly.")
 
+    def test_cmd_with_wrong_argcount_or(self):
+        # $or() requires at least 2 arguments
+        with self.assertRaises(ScriptError):
+            self.parser.eval('$or(0)')
+        try:
+            self.parser.eval('$or(0,1)')
```

Comment:
The positive tests are already covered by `test_cmd_or`, `test_cmd_eq`, … (if there's something missing in them, like testing with more than 2 arguments, it should be added there). I don't see any reason to repeat them here. That would also make this a test that only tests for the case of error, not for the case of success.